src/modules/Builder/features/SaveAndPublish/SaveAndPublish.hooks.ts

```diff
@@ -61,7 +66,7 @@ export const useAppletData = () => {
       };
     }
 
-    return {
+    const result = {
```

Comment:
could you prepare single result forcreation/edition, I see only one point to keep a condition to use appletInfoMocked when isNewApplet is true

Reply:
done